### PR TITLE
Testing Tic-Tac-Toe with Rspec

### PR DESCRIPTION
**Pull request by ElijahTobs**
In this Pull Request,  

1. We initialized "Rspec" in this branch which created basic Rspec files.
2. Test cases were created for **game, player** and **board** classes.
3. Critical methods within these classes were also tested to make sure they return the desired output.

### CODE REVIEW

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required

1. Your specs have puts statements or your classes print to STDOUT, that shouldn't happen, fix that, please.

image

2. You can't assume that the user that is going to test your project has the rspec gem installed on their machine, better specify the project dependencies using a Gemfile, you can find instructions on how to do that in the [bundler documentation](https://bundler.io/v2.0/guides/using_bundler_in_applications.html). Also, remember to put the instructions on how to install the dependencies in your README.

Once you have implemented the required changes please submit another CRR (Code Review Request).

You can find me in slack as **srodrig**, happy coding!

File: README.md [hunk 1]
```diff
@@ -86,6 +89,15 @@ $ git clone https://github.com/acushlakoncept/tic-tac-toe.git
 $ bin/main.rb
 ~~~
 
+Testing
+
+To test the code, run `rspec` from root of the folder using terminal.
+> Rspec is used for the test.
+
+~~~bash
+$ gem install rspec
```
Comment (mosesogwo) [inline README.md]: Installing rspec should come before running rspec. Please reorder this section.

**Review by mosesogwo — APPROVED** [review]
**Hello! Congratulations:+1: Your project is approved:stars:**

You can now merge this PR.

### :heavy_check_mark:

File: README.md (hunk 1 above)
Comment (ElijahTobs) [reply README.md]: > Installing rspec should come before running rspec. Please reorder this section.

Thanks.
We've re-arranged it.

**Pull request merged**